api/ingests_api/src/main/scala/uk/ac/wellcome/platform/storage/ingests/api/responses/CreateIngest.scala

```diff
@@ -50,44 +52,68 @@ trait CreateIngest extends ResponseBase with Logging {
       )
     } else {
       triggerIngestStarter(requestDisplayIngest)
+
     }
   }
 
-  private def createBadRequestResponse(description: String): Route =
+  private def createBadRequestResponse(description: String): Future[Route] = Future {
     complete(
       StatusCodes.BadRequest -> UserErrorResponse(
         contextURL,
         statusCode = StatusCodes.BadRequest,
         description = description
       )
     )
+  }
 
   private def triggerIngestStarter(
     requestDisplayIngest: RequestDisplayIngest
-  ): Route =
-    ingestStarter.initialise(requestDisplayIngest.toIngest) match {
-      case Success(ingest) =>
-        respondWithHeaders(List(createLocationHeader(ingest))) {
+  ): Future[Route] = {
+    val ingest = requestDisplayIngest.toIngest
+
+    val creationResult = for {
+      trackerResult <- ingestTrackerClient.createIngest(ingest)
+      _ <- trackerResult match {
+        case Right(_) => Future.fromTry { unpackerMessageSender.sendT(SourceLocationPayload(ingest)) }
+        case Left(_) => Future.successful(())
+      }
+    } yield trackerResult
```

Comment:
If the ingest gets to the database but not to the unpacker, we treat it as a failed creation, which seems fine to me – the user should try to create it again, or nothing will be stored.

This matches the previous behaviour.